test/tools/libtesteth/TestHelper.h

```diff
@@ -105,7 +111,8 @@ dev::eth::BlockHeader constructHeader(
 	u256 const& _timestamp,
 	bytes const& _extraData);
 void updateEthashSeal(dev::eth::BlockHeader& _header, h256 const& _mixHash, dev::eth::Nonce const& _nonce);
-void executeTests(const std::string& _name, boost::filesystem::path const& _testPathAppendix, boost::filesystem::path const& _fillerPathAppendix, std::function<json_spirit::mValue(json_spirit::mValue const&, bool)> doTests, bool _addFillerSuffix = true);
+void runAllTestsInFolder(test::TestSute const& _testSuite, std::string const& _folder);
```

Comment:
I'm confused, what does `_folder` refer to and what does `_testSuite.folder()` refer to?

Reply:
_folder in runAllTestsInFolder is the folder containing the test files
the folder() of testSuite class is the folder containing all folders with tests for that test suite. 
perhaps testSuite should have  like srcFolder,  destFolder

Reply:
So is `_folder` always a subfolder of `_testSuite.folder()` ? Maybe call the parameter then `_subfolder`  and the function `runTestsInSubfolder` ?
Or at least add a comment about this.

Reply:
It would be nice to see the relation between `runAllTestsInFolder` and `executeTests` from their names.
Like for example naming them `runTestsInSubfolder` & `runTestsInFile` (also you could have `runTestsInSuite` somewhere above)